docs/examples/mlflow/README.md

```diff
@@ -195,108 +195,72 @@ To learn more about how MLServer uses content type parameters, you can check thi
 import requests
 
 inference_request = {
-    "parameters": {
```

Comment:
hmm, we are no longer providing `content_type` in the case of mlflow and using the defaults from `Schema`

what happens if we provide the wrong `content_type`. it is similar to a question I have below in the case there is a mismatch between the two.

Reply:
My understanding is that if the user sends a explicit content type through the request `parameters`, we should then honour that one (as they're trying really hard to use that particular codec). 

I think that should be the case now.